index.js

```diff
@@ -1,134 +1,5 @@
-module.exports = {
-  extends: ['airbnb', 'plugin:prettier/recommended'],
-  env: {
-    es6: true,
-    browser: true,
-  },
-  plugins: ['jsx-a11y'],
-  parserOptions: {
-    ecmaVersion: 9,
-    sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true,
-    },
-  },
-  globals: {},
-  rules: {
-    // ========== CORE ==========
-    // Ref: https://eslint.org/docs/rules/no-underscore-dangle
-    // Airbnb:
-    // 'no-underscore-dangle': ['error', {
-    //   allow: [],
-    //   allowAfterThis: false,
-    //   allowAfterSuper: false,
-    //   enforceInMethodNames: true,
-    // }],
-    //
-    // We use an underscore dangle to mark private and protected variables and methods of a class.
-    // Thats why we need `allowAfterThis`, `allowAfterSuper`, `enforceInMethodNames` to be enabled.
-    'no-underscore-dangle': [
-      'error',
-      {
-        // _id - MongoDB id, convention forced by backend
-        allow: ['_id'],
-        allowAfterThis: true,
-        allowAfterSuper: true,
-        // enforceInMethodNames: true, // available in last version
-      },
-    ],
-
-    // Ref: https://eslint.org/docs/rules/no-unused-expressions
-    // Airbnb:
-    // 'no-unused-expressions': ['error', {
-    //   allowShortCircuit: false,
-    //   allowTernary: false,
-    //   allowTaggedTemplates: false,
-    // }],
-    //
-    // We re-define the rule as we use short circuit syntax: b && b().
-    // Mostly to call disposers of mobx `observe`, `reaction`, `when` etc. Sometimes we define them conditionally and
-    // need call on a component unmount if a disposer exist: dispose && dispose();
-    'no-unused-expressions': [
-      'error',
-      {
-        allowShortCircuit: true,
-        allowTernary: false,
-        allowTaggedTemplates: false,
-      },
-    ],
-
-    // TODO: come back to it when we start to extend prettier recommended
-    // Need to handle a case with Joi Models
-    'newline-per-chained-call': 'off',
-
-    // Ref: https://eslint.org/docs/rules/no-prototype-builtins
-    // Eslint reasoning: https://github.com/eslint/eslint/issues/7071
-    // Airbnb: 'no-prototype-builtins': 'error',
-    // Example (if enable): obj.hasOwnProperty(key) => (should be) Object.prototype.hasOwnProperty.call(obj, key)
-    //
-    // We prefer to get Error and get aware of the wrong usage. It is much better then silently hadle dangerous code.
-    'no-prototype-builtins': 'off',
-
-    // Ref: https://eslint.org/docs/rules/consistent-return
-    // Airbnb: 'consistent-return': 'error'
-    //
-    // TODO: Until we decide, do we want to do `return null` explicitly from function or we stick with
-    // `return undefined`, we disable this rule.
-    'consistent-return': 'off',
-    // ========== END CORE ==========
+const ekoRules = [require.resolve('./rules/base'), require.resolve('./rules/import')];
```

Comment:
Here's a great example of how we can use prettier's comment approach to expand this array

```suggestion
const ekoRules = [
  // Comment about what each of these imports are for
  require.resolve('./rules/base'), 
  require.resolve('./rules/import')
];
```